.env.example

```diff
@@ -39,3 +39,8 @@ DOCKER_IMAGE_BACKEND=survey_backend
 # Other config variables
 DOCKER_IMAGE_CELERYWORKER=survey_celeryworker
 TAG=latest
+
+GOOGLE_AUTHORIZATION_URL=https://accounts.google.com/o/oauth2/v2/auth,
+GOOGLE_TOKEN_URL=https://oauth2.googleapis.com/token
+GOOGLE_CLIENT_ID=233290826498-mfs7tcq5k1vdl2uhmavbcqn2k47dvk2t.apps.googleusercontent.com
```

Comment:
hmm is this something we want committed to a repo?

Reply:
Good call. This is addressed in my next commit. I have removed these URLs from the config entirely and I removed the value for GOOGLE_CLIENT_ID.